### ISSUE TITLE
Integration Tests Failing - Multiple Issue Trackers Enabled

### ISSUE THREAD

**Issue by Shackelford-Arden** [gitlabform/gitlabform#816]
Looks like these are the tests that are failing:

```
FAILED tests/acceptance/standard/test_integrations.py::TestIntegrations::test__if_delete_works - SystemExit: 2
FAILED tests/acceptance/standard/test_integrations.py::TestIntegrations::test__if_push_events_true_works - SystemExit: 2
FAILED tests/acceptance/standard/test_integrations.py::TestIntegrations::test__if_push_events_false_works - SystemExit: 2
FAILED tests/acceptance/standard/test_integrations.py::TestIntegrations::test__if_push_events_change_works - SystemExit: 2
```

Looking at the output, we're getting the same message for all 4 of the tests:

```
"Another issue tracker is already in use. Only one issue tracker service can be active at a time"
```

Looks like this is tied to the fact that both Asana and Redmine are enabled at the same time?

Haven't been able to find any note specifically yet in Gitlab issues/docs that anything in particular has changed. I do see notes [in some older issues](https://gitlab.com/gitlab-org/gitlab/-/issues/328470) though that the constraint on only allowing a single issue tracker to be enabled at a time has been in place for quite some time.

You can see some of this messaging in these PR test results:

https://github.com/gitlabform/gitlabform/actions/runs/10526357975/job/29167036520?pr=814#step:6:802

**Comment by TimKnight-DWP** [comment]
I think let's put a temp-fix in now to just test against Asana, get the pipelines green again, and then keep the Issue open to add back in testing for redmine.

Thoughts @gdubicki @amimas ?

**Comment by gdubicki** [comment]
> I'm assuming we included more than one issue tracker to validate that each of the issue trackers supported the same calls?

No, at least not originally. We just wanted to test some integrations, any of them, one or more. 🤷‍♂️😅

**Comment by Shackelford-Arden** [comment]
haha Fair enough! 😄
